bundles/EcommerceFrameworkBundle/VoucherService/TokenManager/Pattern.php

```diff
@@ -605,13 +605,9 @@ protected function getExportData(array $params)
     }
 
     /**
-     * Removes reservations
-     *
-     * @param int $duration
-     *
-     * @return bool
+     * {@inheritdoc}
      */
-    public function cleanUpReservations($duration = 0)
+    public function cleanUpReservations($duration = 0, $seriesId = null)
     {
         return Reservation::cleanUpReservations($duration, $this->seriesId);
```

Comment:
if we unify this, we at least should add a TODO and issue to also use the $id param in this method call and verify it still works properly.

Reply:
Add TODO here: https://github.com/pimcore/pimcore/pull/9067/commits/054e8f6cbd1b3dca2c1103c0e7e6a13f3bd0a53b